string.md

```diff
@@ -0,0 +1,945 @@
+---
+title: String
```

Comment:
You accidentally generated this file in the root of the project and committed it. Please delete it.